in geister2.py
脱出可能な場合の盤面での事後状態がバグる現象．
脱出フラグは1になるが，謎の場所に青ゴマが存在することになっている
事後状態のみの現象で実際に移動しても問題は起きない．
---(((やはりテストは行うべき)))---
